internal/controller/environments/environments_test.go

```diff
@@ -161,6 +319,23 @@ func TestSync(t *testing.T) {
 						},
 					},
 				},
+				CurrentState: &api.EnvironmentState{
+					Commits: []api.GitCommit{
+						{
+							RepoURL: "fake-url",
+							ID:      "fake-commit",
+						},
+					},
+					Images: []api.Image{
+						{
+							RepoURL: "fake-url",
+							Tag:     "fake-tag",
+						},
+					},
+					Health: &api.Health{
+						Status: api.HealthStateHealthy,
+					},
+				},
```

Comment:
This is unrelated to the PR, but with these lines missing there was a branch of this function that wasn't being covered by tests. It seemed sensible to fix it while already working on these test cases.